docs/Technical Guides/EnablingShopperProfiles.md

```diff
@@ -0,0 +1,15 @@
+
```

Comment:
General comment on the Technical Guides.  They seem to be copies of the Demo Guide that can be launched from within the web-ui.  Therefore, we need to maintain the information in two places.  Is it possible to pull this content from the demo guide?

Reply:
This is correct, I haven't found a fully automated way to do it. I see options:

- Remove the demo guide from inside the demo itself (and replace with a link to this documentation in the header or with an iframe)
- Automate the demo guide generation into 
  - the vue components
  - this mkdocs doc
- Don't include these docs in the mkdocs and only in the demo 

I prefer the first option but we would benefit from a bit of a UI refresh (a better header, remove/change the demo guide button, ...)

to be discussed?